### PR TITLE
[KIECLOUD-332] image naming convention changes

### PR DESCRIPTION
**Pull request by sutaakar**
https://issues.jboss.org/browse/KIECLOUD-332

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[KIECLOUD-XYZ] Subject`, `[RHDM-XYZ] Subject` or `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by sutaakar** [comment]
@errantepiphany Can you please review?

File: tests/features/common/kie-common.feature [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@rhdm-7/rhdm76-kieserver-openshift  @rhpam-7/rhpam76-kieserver-openshift @rhdm-7/:wq!-decisioncentral-openshift @rhpam-7/rhpam76-businesscentral-openshift @rhpam-7/rhpam76-businesscentral-monitoring-openshift
+@rhdm-7/rhdm-kieserver-rhel8  @rhpam-7/rhpam-kieserver-rhel8 @rhdm-7/:wq!-decisioncentral-openshift @rhpam-7/rhpam-businesscentral-rhel8 @rhpam-7/rhpam-businesscentral-monitoring-rhel8
```
Comment (errantepiphany) [inline tests/features/common/kie-common.feature]: Accidental vim commands included... ":wq!" should be "rhdm".

Comment (sutaakar) [reply tests/features/common/kie-common.feature]: Fixed

**Comment by sutaakar** [comment]
I think it is ok to retag the 7.5.0.

**Pull request merged**